src/main/java/org/jasig/portlet/newsreader/dao/HibernateNewsStore.java

```diff
@@ -136,8 +136,7 @@ public void storeNewsConfiguration(NewsConfiguration configuration) {
         try {
 
             String query = "from PredefinedNewsDefinition def "
-                    + "where :setId not in (select config.newsSet.id "
-                    + "from def.userConfigurations config) ";
+                    + "where not exists (from def.userConfigurations config where config.newsSet.id = :setId) ";
```

Comment:
:art: 

> Always use uppercase for the reserved keywords like SELECT and WHERE.

ref: http://www.sqlstyle.guide/#reserved-words

Reply:
Since this is not SQL, but rather HQL, I was going off of ...

> This manual uses lowercase HQL keywords. Some users find queries with uppercase keywords more readable, but this convention is unsuitable for queries embedded in Java code.

from https://docs.jboss.org/hibernate/orm/3.3/reference/en-US/html/queryhql.html.

Reviewing http://www.sqlstyle.guide/, they recommend not using camelcase but rather underscores. So there is a significant difference between them.

Do you still think that overlapping SQL keywords should be uppercase? I am on the fence in that particular one.

Reply:
I'd lean mildly toward using uppercase for overlapping keywords.
But would be okay leaving it as is.

Reply:
Sounds good. It would make the HQL easier to read.